web.py

```diff
@@ -85,13 +85,19 @@ def fetch_kle_json(gist_id):
             logging.info('Using cache file %s (%s < 30)', cache_file, file_age)
             return copen(cache_file, encoding='UTF-8').read()
         else:
-            headers['If-Modified-Since'] = strftime('%a, %d %b %Y %H:%M:%S %Z', localtime(file_stat.st_mtime))
-            logging.warning('Adding If-Modified-Since: %s to headers.', headers['If-Modified-Since'])
+            headers['If-Modified-Since'] = strftime(
+                '%a, %d %b %Y %H:%M:%S %Z',
+                localtime(file_stat.st_mtime))
+            logging.warning(
+                'Adding If-Modified-Since: %s to headers.',
+                headers['If-Modified-Since'])
 
```

Comment:
Please don't reformat these. Local style is to use lines that are as long as they need to be.